requirements.txt

```diff
@@ -18,3 +18,4 @@ pytest
 coverage
 pytest-cov
 pygrib~=2.1.4
+geopy
```

Comment:
Someone else can probably provide better guidance on this, but should we be specifying a version of the `geopy` package?

Reply:
I will look into that later.

Reply:
Specifying a version number isn't strictly necessary, but we'll want to add `geopy` to the `Pipfile`/`Pipfile.lock` so that all functions work after building the environment using `pipenv`.